ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -79,6 +79,13 @@ $font-weight-heading: bold !default;
 $max-width-page: 1200px !default;
 $line-height-base: 1.5em !default;
 
+/*
+  Spinner
+  Declare this variable in the policy to define a spinner.
+  Use a resource URL which points to a gif spinner.
+ */
+$spinner-url: "" !default;
```

Comment:
What does happen when the url is not declared?
I think the image will just not appear, right?
This would be ok, but we should not have an error, since the project may not install a package using the spinner and thus have no use for that.